packages/web/e2e/tests/portfolio.wallet.spec.ts

```diff
@@ -58,33 +58,46 @@ test.describe("Test Portfolio feature", () => {
   });
 
   test("User should be able to see native balances", async () => {
+    await portfolioPage.searchForToken("OSMO");
```

Comment:
Nit: we could probably put these in a loop to minimize cognitive overhead stemming from code duplication

Reply:
agree, I just need to do it. Moved to parameteried tests approach.